CONTRIBUTING.md

```diff
@@ -15,10 +15,18 @@
 * `cos-alerter`
 
 ## Run With Docker
-
-* Copy the contents of `cos-alerter-default.yaml` to `cos-alerter.yaml` and fill in with appropriate values.
-* `docker build . -t cos-alerter`
-* `docker run -p 8080:8080 --mount type=bind,source="$(pwd)"/cos-alerter.yaml,target=/etc/cos-alerter.yaml,readonly -it cos-alerter`
+Prepare the image:
+```shell
+rockcraft pack
+skopeo --insecure-policy copy oci-archive:cos-alerter_0.2.0_amd64.rock docker-daemon:cos-alerter:0.2.0
+```
+
+Run:
+```shell
+cp cos-alerter.sample.yaml cos-alerter.yaml
+# Update cos-alerter.yaml with appropriate values
+docker run -p 8080:8080 --rm --mount type=bind,source="$(pwd)"/cos-alerter.yaml,target=/etc/cos-alerter.yaml,readonly -it cos-alerter:0.2.0
+```
```

Comment:
I would explain a little further these steps.

Reply:
Documented [here](https://github.com/canonical/rockcraft/pull/239) and [here](https://discourse.charmhub.io/t/containerize-a-pypi-package-using-rockcraft/10224). Let's wait for the dust to settle and maybe we could just link there instead of repeating here.